st2common/st2common/models/db/execution.py

```diff
@@ -128,31 +128,6 @@ def mask_secrets(self, value):
         result["parameters"] = mask_secret_parameters(
             parameters=result.get("parameters", {}), secret_parameters=secret_parameters
         )
-
-        if "parameters" in liveaction:
-            liveaction["parameters"] = mask_secret_parameters(
-                parameters=liveaction["parameters"], secret_parameters=secret_parameters
-            )
-
-            if liveaction.get("action", "") == "st2.inquiry.respond":
-                # Special case to mask parameters for `st2.inquiry.respond` action
```

Comment:
Where does this handling happen now?

Reply:
That makes sense. Please move the inquiry masking into the live action model since that is what the API will be retrieving.